plugin/wavesurfer.regions.js

```diff
@@ -424,12 +424,12 @@ WaveSurfer.Region = {
 
                     // Drag
                     if (my.drag && drag) {
```

Comment:
sorry, this will not break the events to the region as I said before. It just won't define drag() or resize() as a function.